src/Resources/ResourceManager/Components/Constants.cs

```diff
@@ -30,9 +30,14 @@ public static class Constants
         public static readonly string MicrosoftAuthorizationNamespace = "Microsoft.Authorization";
 
         /// <summary>
-        /// The <c>Microsoft.Management</c> namespace.
+        /// The management group type.
         /// </summary>
-        public static readonly string MicrosoftManagementNamespace = "Microsoft.Management";
+        public static readonly string ManagementGroupType = "/providers/Microsoft.Management/managementGroups/";
+
+        /// <summary>
+        /// The management group type name.
+        /// </summary>
+        public static readonly string ManagementGroupTypeName = "managementGroups";
```

Comment:
>ManagementGroupTypeName [](start = 38, length = 23)

nit - from the existing naming convention, this should be called `ManagementGroups`

Reply:
refactored.